prog/vm/github_runner.rb

```diff
@@ -220,6 +220,8 @@ def setup_info
       # ubicloud/cache package which forked from the official actions/cache package, sends requests to UBICLOUD_CACHE_URL using this token.
       echo "UBICLOUD_RUNTIME_TOKEN=#{vm.runtime_token}
       UBICLOUD_CACHE_URL=#{Config.base_url}/runtime/github/" | sudo tee -a /etc/environment
+      echo '{"ipv6": true, "fixed-cidr-v6": "#{vm.ephemeral_net6.nth(2)}:dc/124", "experimental": false, "dns": ["#{vm.nics.first.private_subnet.net4.nth(2)}", "fd00:0b1c:100d:53::"]}' | sudo tee /etc/docker/daemon.json
```

Comment:
I wonder if you should break out a JSON.generate(...).shellescape. I suppose we could go with this if we never anticipate ever wanting to change it again, because a typo would cause havoc in short order given where it is.